### ISSUE TITLE
waterfox-classic fails to build with rust-1.45.0

### ISSUE THREAD

**Issue by Polynomial-C** [MrAlex94/Waterfox#1663]
```
   Compiling gkrust v0.1.0 (/var/tmp/portage/www-client/waterfox-classic-2020.07/work/Waterfox-2020.07-classic/toolkit/library/rust)
error: options `-C embed-bitcode=no` and `-C lto` are incompatible

error: could not compile `gkrust`.
```
The following patch should fix that:

```
--- Waterfox-2020.07-classic/config/rules.mk
+++ Waterfox-2020.07-classic/config/rules.mk
@@ -864,12 +864,6 @@
 # These flags are passed via `cargo rustc` and only apply to the final rustc
 # invocation (i.e., only the top-level crate, not its dependencies).
 cargo_rustc_flags = $(CARGO_RUSTCFLAGS)
-ifndef DEVELOPER_OPTIONS
-ifndef MOZ_DEBUG_RUST
-# Enable link-time optimization for release builds.
-cargo_rustc_flags += -C lto
-endif
-endif
 
 # Cargo currently supports only two interesting profiles for building:
 # development and release.  Those map (roughly) to --enable-debug and
```

**Comment by hawkeye116477** [comment]
@Polynomial-C This should be fixed by #1675 now.

**Comment by Polynomial-C** [comment]
Thanks. This seems to be fixed with 2020.07.1
